.github/workflows/golangci-lint.yaml

```
@@ -0,0 +1,28 @@
+# https://github.com/golangci/golangci-lint-action
+# act --container-architecture linux/amd64 --workflows .github/workflows/golangci-lint.yaml
+name: golangci-lint
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+permissions:
+  contents: read
+jobs:
+  golangci:
+    strategy:
+      matrix:
+        go: ['1.20','1.19']
+#        bug in goimports on windows-latest
+        os: [ubuntu-latest, macos-latest]
```

Why do we need to burn macOS minutes on this?
```suggestion
        os: [ubuntu-latest]
```